recipes/polybeat/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "polybeat" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 17fc46568071985446294b703696697778954f2a8ee62577563e46a6dc3e9a68
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+  run:
+    - python >=3.7
+    - manim
+
+test:
+  imports:
+    - polybeat
+  requires:
+    - pip
+    - manim
```

Comment:
This is probably not needed, since you have it in the `run` section as well.

Reply:
This comment still applies.

Reply:
hi @xhochy , I've tried deleting this code, but it failed the test. Could you please give me more specific instructions?

Reply:
Please remove it and let CI fail. Then I can look into it and give recommendations.

Reply:
@xhochy check please